extensions/blocks/revue/revue.php

```diff
@@ -35,16 +35,18 @@ function jetpack_render_revue_block( $attributes ) {
 	$last_name_placeholder  = jetpack_get_revue_attribute( 'lastNamePlaceholder', $attributes );
 	$last_name_show         = jetpack_get_revue_attribute( 'lastNameShow', $attributes );
 	$url                    = sprintf( 'https://www.getrevue.co/profile/%s/add_subscriber', $attributes['revueUsername'] );
```

Comment:
I know that's not part of this PR, but I think we need to escape this username. I had missed that before, sorry!

Reply:
We escape the URL a few lines later though: `action="<?php echo esc_url( $url ); ?>"`
Do I still need to escape the username here too? 🤔

Reply:
True. I just realized others are having the same conversation elsewhere. :)
https://github.com/Automattic/jetpack/pull/14590#discussion_r376060874

Let's keep things as they are in here then. Ideally, maybe we should validate the username to avoid the use of characters like `/` for example.